zerver/lib/webhooks/git.py

```diff
@@ -163,6 +190,22 @@ def get_commits_content(commits_data, is_truncated=False):
         ).replace('  ', ' ')
     return commits_content.rstrip()
 
+
 def get_short_sha(sha):
     # type: (Text) -> Text
     return sha[:7]
+
+
+def get_all_committers(commits_data):
+    # type: (List[Dict[str, Any]]) -> Dict[str, int]
+    committers = defaultdict(int)  # type: Dict[str, int]
+    for commit in commits_data:
+        committers[commit['name']] += 1
+    committers_names = list(sorted(committers.keys()))  # type: List[str]
```

Comment:
I am sorry I thought i just needed to show committers along with their number of commits and not in the decreasing order of number of commits.
I'll make the change ;)